docs/extend/plugins_graphdriver.md

```diff
@@ -5,8 +5,8 @@ graph driver for use with Docker engine. This is an alternative to using the
 built-in storage drivers, such as aufs/overlay/devicemapper/btrfs.
```

Comment:
What do you mean about metadata added? In the docs repo we will need to add a TOC reference for this file, but I don't understand what else you mean...